src/cplscheme/SerialCouplingScheme.cpp

```diff
@@ -106,11 +90,11 @@ void SerialCouplingScheme::exchangeInitialData()
       // The second participant sends the initialized data to the first participant
       // here, which receives the data on call of initialize().
       sendData(getM2N(), getSendData());
-      receiveAndSetTimeWindowSize();
-      // This receive replaces the receive in initialize().
-      receiveData(getM2N(), getReceiveData());
-      checkDataHasBeenReceived();
     }
+    receiveAndSetTimeWindowSize();
+    // This receive replaces the receive in initialize().
```

Comment:
We can probably remove this comment.

Reply:
Done 236449928392a750510803a148b1202ec8b6520a.